tests/blockchain.rs

```diff
@@ -410,69 +409,700 @@ fn load_external_contract_code() {
     }
 }
 
-#[test]
-fn can_read_state_from_initial_storage_slots() {
-    // the initial key and value pair for the contract
-    let key = Hasher::hash(b"initial key");
-    let value = [128u8; 32].into();
+fn ldc_reason_helper(cmd: Vec<Opcode>, expected_reason: PanicReason, should_patch_jump: bool) {
```

Comment:
nit: `#[test]` directive is missing

Reply:
this isn't a test, it is a helper function used by a test. Is there a way to mark something as FOR a test but not to be executed independently?

Reply:
`#[cfg(test)]` makes a function only available when running tests

Reply:
although, since this function is defined inside the `tests` directory, it will only ever be available within a testing context. Cargo ignores anything under `#[cfg(test)]`, `#[test]` or in the `/tests` dir when publishing a crate.

Reply:
It would be cleaner without the annotations IMO, they aren't needed here. While not a big deal, we probably don't want this to become a pattern repeated throughout the codebase.

Reply:
Agreed; if it isn't a test case, then further directives aren't required